charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-daemonset.yaml

```diff
@@ -24,8 +24,8 @@ spec:
 {{ toYaml .Values.labels | indent 8 }}
         networking.gardener.cloud/to-dns: allowed
         networking.gardener.cloud/to-runtime-apiserver: allowed
-        networking.resources.gardener.cloud/to-logging-tcp-metrics: allowed
-        networking.resources.gardener.cloud/to-all-shoots-loki-tcp-metrics: allowed
+        networking.resources.gardener.cloud/to-logging-tcp-3100: allowed
```

Comment:
Why you don't name to something like "ingest"? We already discussed that the current name is wrong - port named "metrics" that is actually used for pushing logs to loki.